### PR TITLE
Enable HttpFS

### PR DESCRIPTION
**Pull request by jack-altiscale**
- Added support for HttpFS (over HTTP)

### CODE REVIEW

File: dev-support/docker-compose/scripts/start-namenode.sh [hunk 1]
```diff
@@ -5,4 +5,5 @@ if [ $FORMAT_NAMENODE == true ]; then
 fi
 
 /opt/hadoop/hadoop-dist/target/hadoop-2.7.4/sbin/hadoop-daemon.sh --config /etc/hadoop --script hdfs start namenode
+/opt/hadoop/hadoop-dist/target/hadoop-2.7.4/sbin/httpfs.sh start
```
Comment (zoran-altiscale) [inline dev-support/docker-compose/scripts/start-namenode.sh]: It's better to start httpfs in its own container. Common case is to have a pool of httpfs replicas per cluster.

File: dev-support/docker-compose/docker-compose.yml [hunk 2]
```diff
@@ -23,6 +23,8 @@ namenode:
     - "50070:50070"
     - "50470:50470"
     - "8020:8020"
+    - "14000:14000"
+    - "14001:14001"
```
Comment (zoran-altiscale) [inline dev-support/docker-compose/docker-compose.yml]: no need to export this port.

File: dev-support/docker-compose/scripts/start-httpfs.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+sleep 5
```
Comment (zoran-altiscale) [inline dev-support/docker-compose/scripts/start-httpfs.sh:3]: why are you sleeping 5 seconds? Does start script fails if namenode hasn't started yet?

**Review by zoran-altiscale — APPROVED** [review]
lgtm++

File: dev-support/docker-compose/scripts/start-httpfs.sh (hunk 3 above)
Comment (jack-altiscale) [reply dev-support/docker-compose/scripts/start-httpfs.sh:3]: No, it is just for the docker-compose log output to be more organized. I plan to circle back to this at a later time

**Pull request merged**